helm/botkube/values.yaml

```diff
@@ -307,3 +307,16 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+rbac:
+  create: true
+  rules:
+    - apiGroups: ["*"]
+      resources: ["*"]
+      verbs: ["get", "watch", "list"]
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: test
```

Comment:
shouldn't this be not set by default so fullname is used?

Reply:
Thanks, was just added to **test** in local :)